feedback/responses.py

```diff
@@ -0,0 +1,7 @@
+feedback_created_201 = {
+    "message" : "Feedback posted successfully"
+}
+
+feedback_invalid_400 = {
+    "message" : "error_msg"
```

Comment:
Specify the error messages that can be returned as response from the API. The `error_msg` is a variable.

Reply:
Actually it could be any validation error, there are many options, should I specify any one of them here?

Reply:
Try specifying most of them in a list 👍

Reply:
Done sir